src/commands/vote/vote.command.ts

```diff
@@ -93,6 +93,13 @@ export class VoteCommand implements CommandClass {
             case 'staff':
             case 'ducks':
             case 'both':
+              if (detectOutsideStaff(message.author) === 'admin') {
```

Comment:
I think we could make this cleaner by allowing a list of roles to be provided instead, so the options would become 'staff', 'ducks', and 'cats'. Since cats can only do these in their channels, I'm okay if they have the ability to include ducks/staff in the pings, we don't mind, and eventually, the ducks may be involved anyway.